inc/saihostintf.h

```diff
@@ -330,8 +350,7 @@ typedef enum _sai_hostif_trap_attr_t
      * @brief Trap action
      *
      * @type sai_packet_action_t
-     * @flags CREATE_AND_SET
-     * @default SAI_PACKET_ACTION_FORWARD
+     * @flags CREATE_AND_SET | MANDATORY_ON_CREATE
      */
     SAI_HOSTIF_TRAP_ATTR_PACKET_ACTION,
```

Comment:
I think this attribute should move. The action should be definable based on per ingress port etc. So if same trap is applied on different ports, the action could be different.